.github/workflows/pr.yml

```diff
@@ -60,267 +24,29 @@ jobs:
         run: brew install ninja
       - name: Setup Watchman
         run: brew install watchman
-      - name: NPM Install (Playground)
-        run: npm install
-        working-directory: ./Apps/BRNPlayground
-
-      - name: Build Windows Bundle
-        run: npm run build:android
-        working-directory: ./Apps/BRNPlayground
-
-      - name: NPM Install (Binary Package)
-        run: npm install
-        working-directory: ./Package
-      - name: Setup Java
-        uses: actions/setup-java@v3
-        with:
-          distribution: 'temurin'
-          java-version: '17'
-          check-latest: true
-      - name: Gulp (Android)
-        run: npx gulp buildAndroidRNTA
-        working-directory: ./Package
-        env:
-          JAVA_HOME: ${{ env.JAVA_HOME_11_X64 }}
-  
-  build-iOS:
-    runs-on: macos-latest
-    steps:
-      - name: Checkout Repo
-        uses: actions/checkout@v2.3.3
-        with:
-          submodules: 'recursive'
-      - name: Setup Node 16
-        uses: actions/setup-node@v3
-        with:
-          node-version: 16
-      - name: NPM Install (Playground)
-        run: npm install
-        working-directory: ./Apps/Playground
-      - name: NPM Install (React Native ${{ env.BRN_Version }})
-        run: npm run select --reactNative ${{ env.BRN_Version }}
-        working-directory: ./Apps/Playground
-      - name: NPM Install (Binary Package)
-        run: npm install
-        working-directory: ./Package
-      - name: Gulp (iOS)
-        run: npx gulp buildIOS
-        working-directory: ./Package
-
-  build-iOS-rnta:
-    runs-on: macos-latest
-    steps:
-      - name: Checkout Repo
-        uses: actions/checkout@v2.3.3
-        with:
-          submodules: 'recursive'
-      - name: Setup Watchman
-        run: brew install watchman
-  
-      - name: NPM Install (Playground)
+      - name: Patch BRNPlayground for RN ${{ matrix.react-native-version }}
+        run: git apply rnta-${{ matrix.react-native-version }}.patch --allow-empty
```

Comment:
Can we do this as part of a script that is easy to run locally? Like the `npm run select 0.71` we currently have?